src/libs/antares/study/area/list.cpp

```diff
@@ -305,6 +305,33 @@ static bool AreaListSaveToFolderSingleArea(const Area& area, Clob& buffer, const
         buffer.clear() << folder << SEP << "input" << SEP << "renewables" << SEP << "series";
         ret = area.renewable.list.saveDataSeriesToFolder(buffer) && ret;
     }
+
+    // Short term storage
+
+    // create the main input/st-storage/clusters directory
+    buffer.clear() << folder << SEP << "input" << SEP << "st-storage" << SEP << "clusters"
+        << SEP << area.id;
+    IO::Directory::Create(buffer);
+
+    // create empty list.ini for this area
+    buffer.clear() << folder << SEP << "input" << SEP << "st-storage" << SEP << "clusters"
+        << SEP << area.id << SEP << "list.ini";
+    IO::File::CreateEmptyFile(buffer);
```

Comment:
I think you don't actually need to create empty list.ini here. You could do it in the `STStorageInput::saveToFolder` instead.

The same goes for all other directory creations.